src/scene/data.ts

```diff
@@ -44,6 +47,61 @@ export function saveAsJSON(
   );
 }
 
+export async function exportToShortlink(
+  elements: readonly ExcalidrawElement[]
+) {
+  const serialized = JSON.stringify({
+    version: 1,
+    source: window.location.origin,
+    elements: elements.map(({ shape, ...el }) => el)
+  });
+
+  // Generating a random unique shortcode
+  // https://stackoverflow.com/questions/6248666/
+  const firstPart = (Math.random() * 46656) | 0;
+  const secondPart = (Math.random() * 46656) | 0;
+  const shortcode =
+    ("000" + firstPart.toString(36)).slice(-3) +
+    ("000" + secondPart.toString(36)).slice(-3);
+
+  const response = await fetch(
+    JSONSTOREIO_SHORTLINK_ENDPOINT + "/" + shortcode,
+    {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: serialized
+    }
+  );
+  const json = await response.json();
+  if (json.ok) {
+    const url = new URL(window.location.href);
+    url.searchParams.append("share", shortcode);
```

Comment:
Can you add a prefix like “js” for json store so that if we decide to use another service at some point we can figure out from where it’s coming from?